solr/core/src/java/org/apache/solr/core/SchemaCodecFactory.java

```diff
@@ -151,8 +151,9 @@ public Codec getCodec() {
   }
 
   /**
-   * This class exists because Lucene95HnswVectorsFormat's getMaxDimensions method is final and we
-   * need to workaround that constraint to allow more than the default number of dimensions
+   * This class exists because {@link Lucene99HnswVectorsFormat#getMaxDimensions(String)} method is
+   * final and we need to workaround that constraint to allow more than the default number of
```

Comment:
Actually the method is not (no longer?) final: https://github.com/apache/lucene/blob/releases/lucene/9.10.0/lucene/core/src/java/org/apache/lucene/codecs/lucene99/Lucene99HnswVectorsFormat.java#L220